docs/architecture/ADR-003-borrow-assets.md

```diff
@@ -102,6 +134,9 @@ collateralSettingPrefix | lengthPrefixed(borrowerAddress) | tokenDenom = true/fa
 
 // and the amount of collateral deposited for each uToken:
 collateralAmountPrefix | lengthPrefixed(borrowerAddress) | tokenDenom = sdk.Int
+
+// max token collateral utilization setings
+maxCollateralUtilizationPrefix | token = bigEndian(uint32)
```

Comment:
Is there a reason not to store it in the `Token` registry with the other per-token parameters?